### PR TITLE
Adding s3_origin_config to docs in parameters and return values (from changes in PR #39)

### PR DESCRIPTION
**Pull request by landphil**
https://github.com/ansible-collections/community.aws/pull/39

Adding `s3_origin_config` to docs in parameters and return values

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by markuman** [comment]
@landphil thx for your first PR to this collection.  
You need also `changelog/fragments/` file. When you add it, it will automatically kick the CI again, because the failed task is not related with your PR.

File: plugins/modules/cloudfront_distribution.py [hunk 1]
```diff
@@ -149,9 +149,16 @@
         s3_origin_access_identity_enabled:
           description:
             - Use an origin access identity to configure the origin so that viewers can only access objects in an Amazon S3 bucket through CloudFront.
-            - Will automatically create an Identity for you.
+            - Will automatically create an Identity for you if no I(s3_origin_config) is specified.
             - See also U(https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/PrivateContent.html).
           type: bool
+        s3_origin_config:
+          description: Specify origin access identity for S3 origins
```
Comment (tremble) [inline plugins/modules/cloudfront_distribution.py:156]:
```suggestion
          description: Specify origin access identity for S3 origins.
```

File: plugins/modules/cloudfront_distribution.py [hunk 2]
```diff
@@ -149,9 +149,16 @@
...
+          type: dict
+          suboptions:
+            origin_access_identity:
+              description: Existing origin access identity in the format C(origin-access-identity/cloudfront/OID_ID)
```
Comment (tremble) [inline plugins/modules/cloudfront_distribution.py:160]:
```suggestion
              description: Existing origin access identity in the format C(origin-access-identity/cloudfront/OID_ID).
```

File: plugins/modules/cloudfront_distribution.py [hunk 3]
```diff
@@ -1275,6 +1282,15 @@
...
           type: str
           sample: ''
+        s3_origin_config:
+          description: Origin access identity configuration for S3 Origin.
+          returned: when s3_origin_access_identity_enabled is true
+          type: dict
+          contains:
+            origin_access_identity:
+              type: str
+              description: The origin access id as a path
```
Comment (tremble) [inline plugins/modules/cloudfront_distribution.py:1292]:
```suggestion
              description: The origin access id as a path.
```

**Review by tremble — CHANGES_REQUESTED** [review]
niggles

**Review by tremble — APPROVED** [review]

**Comment by landphil** [comment]
> @landphil thx for your first PR to this collection.
> You need also `changelog/fragments/` file. When you add it, it will automatically kick the CI again, because the failed task is not related with your PR.
Hi Markus- The developer docs say you don't add a changelog entry for documentation changes:

https://docs.ansible.com/ansible/latest/community/development_process.html#making-your-pr-merge-worthy

I can get one in there if you really want it, though.

Thanks all for getting this merged- I was about 10 minutes into modifying the module to add the functionality I needed, when I realized it was already there and undocumented!

**Comment by tremble** [comment]
For a pure docs update I wouldn't worry about the changelog.  I've kicked off the final test/merge workflow.

Thanks for your contribution.

**Comment by alinabuzachis** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Pull request merged**